pkg/reconciler/util.go

```diff
@@ -0,0 +1,45 @@
+package reconciler
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	networkingpkg "knative.dev/networking/pkg"
+	"knative.dev/networking/pkg/apis/networking"
+	netv1alpha1 "knative.dev/networking/pkg/apis/networking/v1alpha1"
+	"knative.dev/pkg/logging"
+)
+
+func GetHTTPOption(ctx context.Context, networkConfig *networkingpkg.Config, annotations map[string]string) (netv1alpha1.HTTPOption, error) {
```

Comment:
nit: could you please add a comment for this function?

Reply:
done